### ISSUE TITLE
bug: Not able to intercept HTTP on localhost

### ISSUE THREAD

**Issue by gomstory** [requestly/requestly#674]
### Describe your issue?

When running localhost (port 5000) and inspect HTTP on Requestly, the error shows
`PROXY_TO_SERVER_REQUEST_ERROR: Error: connect ECONNREFUSED 127.0.0.1:5000`

### Repro steps

Steps:
1. Run local Angular Application 
2. Open Requestly and connect app with Google Chrome
3. type `localhost:5000` on the URL bar.
4. the error shows `PROXY_TO_SERVER_REQUEST_ERROR: Error: connect ECONNREFUSED 127.0.0.1:5000` 
5. if I close Requestly, localhost running perfectly.

### What Requestly tool were you using?

- [X] Extension
- [X] Desktop-App
- [ ] Android SDK
- [ ] Selenium

### Your Environment

Mac OS 13

### Requestly Version

1.5.0

**Comment by wrongsahil** [comment]
@gomstory I tried this with a local react server running on localhost:3000. Seems to be working fine for me. Can you share your Angular Application settings so that i can try running the same on my system?

**Comment by gomstory** [comment]
@wrongsahil Thanks for your reply. Is this require the certificate to be installed or any other setting?

As of now, I am using Chrome extension as a workaround even though I can't see response in Network tab in Dev tool.
But it's working.

**Comment by wrongsahil** [comment]
No, Certificate installation is not required while connecting to an isolated chrome instance. 

Chrome extension should work similar to desktop app. Only thing extension can't do is to modify requests other than fetch and xhr. So if your request is xhr/fetch, then the extension will work for you.

We got one more user reporting the same issue with angular and desktop app. Let me look it. I'll keep you updated here on the issue

**Comment by ValeryG** [comment]
HI! any updates on the use? I am getting the same when using desktop version of Requestify , vite app running on port 8080  and chrome as a browser. ```

**Comment by wrongsahil** [comment]
@ValeryG We didn't had any success with this last time. We'll try to investigate this issue again this week. 
Thanks for the additional details. This will certainly help a lot in debugging

**Comment by Aarushsr12** [comment]
Hey @ValeryG @gomstory , we have released a fix for localhost requests not working on desktop app. please try this again, if you are facing still, please report here, thanks! 
Fix: https://github.com/requestly/requestly-desktop-app/pull/276
